image/src/app/weeve/egress.py

```diff
@@ -18,18 +19,27 @@ def send_data(data: json, timestamp=time.time()) -> bool:
     """
 
     try:
+        headers = {}
+        if APPLICATION['AUTHENTICATION_REQUIRED'] == 'yes' and APPLICATION['ACCESS_TOKEN'] != '':
+            headers.update({"Authorization": f"{APPLICATION['ACCESS_TOKEN']}"})
+        if APPLICATION['AUTHENTICATION_API_KEY'] != '':
+            headers.update({'x-api-key': APPLICATION['AUTHENTICATION_API_KEY']})
         # URL Convetion 1
         if not WEEVE['EGRESS_URL']:
-            resp = post(url=f"{WEEVE['EGRESS_SCHEME']}://{WEEVE['EGRESS_HOST']}:{WEEVE['EGRESS_PORT']}/{WEEVE['EGRESS_PATH']}", json=data)
+            resp = post(
+                url=f"{WEEVE['EGRESS_SCHEME']}://{WEEVE['EGRESS_HOST']}:{WEEVE['EGRESS_PORT']}/{WEEVE['EGRESS_PATH']}",
+                json=data,
+                headers=headers,
+            )
         # URL Convetion 2
         else:
-            resp = post(url=f"{WEEVE['EGRESS_URL']}", json=data)
-        
+            resp = post(url=f"{WEEVE['EGRESS_URL']}", json=data, headers=headers)
+
```

Comment:
We don't need this changes. Please, revert it.

Reply:
done.